src/studio/ui/QGrid.java

```diff
@@ -35,8 +35,13 @@ public int getRowCount() {
     static class MYJTable extends JTable {
         public MYJTable(TableModel m) {
             super(m);
+            Font tableFont = Config.getInstance().getTableFont();
+            setFont(tableFont);
+            setRowHeight((int)Math.round(getRowHeight() * (tableFont.getSize() / Config.DEFAULT_FONT_SIZE)));
```

Comment:
I think this one would work better:
`setRowHeight(getFontMetrics(tableFont).getHeight());`

Reply:
Will try it